### PR TITLE
fix(helm): Check prior vendoring of helm charts

### PR DESCRIPTION
**Pull request by justinwalz**
Fixes https://github.com/grafana/tanka/issues/401

### CODE REVIEW

File: pkg/helm/charts.go [hunk 1]
```diff
@@ -199,3 +206,14 @@ func parseReq(s string) (*Requirement, error) {
 		Version: *ver,
 	}, nil
 }
+
+// parseReqName parses a name from a string of the format `repo/name` or `name`
+func parseReqName(s string) string {
```
Comment (sh0rez) [inline pkg/helm/charts.go:211]: It seems like `parseReqName` is only ever called with an argument of the form `name@repo` as `r.Chart` looks like this.

Any reason this function also handles plain `name`?

**Review by sh0rez — COMMENTED** [review]
Thanks for this awesome contribution! Got one small question, then this is good to go :)

File: pkg/helm/charts.go (hunk 1 above)
Comment (justinwalz) [reply pkg/helm/charts.go:211]: I'm just being defensive in the case a chart name didn't have a `repo/` prefix, I'm not sure how the helm community does it across the board.

Comment (justinwalz) [reply pkg/helm/charts.go:211]: @sh0rez Does that make sense? I'm happy to simplify it if that case won't happen. Just let me know.

Comment (Duologic) [reply pkg/helm/charts.go:211]: The use of `repo/name` promotes making a clear distinction which Chart one refers to, I would be in favor of enforcing this.

Comment (justinwalz) [reply pkg/helm/charts.go:211]: Gotcha. I removed the check.

Comment (sh0rez) [reply pkg/helm/charts.go:211]: Thanks @justinwalz!

**Review by sh0rez — APPROVED** [review]
LGTM :rocket:

**Pull request merged**